cli/config/presets.go

```diff
@@ -0,0 +1,80 @@
+package config
+
+import "sigs.k8s.io/yaml"
+
+const (
+	PresetDemo   = "demo"
+	PresetSecure = "secure"
+)
+
+// Presets is a map of pre-configured helm values.
+var Presets = map[string]interface{}{
+	PresetDemo:   Convert(demo),
+	PresetSecure: Convert(secure),
+}
+
+const demo = `
+global:
+  name: consul
+connectInject:
+  enabled: true
+server:
+  replicas: 1
+  bootstrapExpect: 1
+`
+
+// TODO: I don't know why the following hangs for me.
```

Comment:
Could these presets be re-enabled again?

Reply:
Yes, I'll test why it was hanging for Saad.